README.rst

```diff
@@ -32,6 +32,35 @@ Which runs all available tests, or run only a specific test::
 
 The above lines are entirely up to the user to modify, and will allow them to choose in which way they want to use the tests.
 
+Running Tests Locally
+#####################
+
+You can start running tests locally by adding the mongoLT (Local Test) hosts with the add-etc-local-hosts.sh bash script as follows::
+
+  $ ./local_setup/add-etc-local-hosts
+  
+Then you can add the required environment variables with the following add-persistent-env-variables.sh bash script::
+
+  $ sudo ./local_setup/add-env-variables
+
+Subsequently, the docker-compose.local.yml file can be used with the following command to run all of the required docker containers::
+
+  $ docker-compose -f docker-compose.local.yml up -d
+
+To make sure that the DB connectivity is functional run::
+
+  $ python scripts/wait_for_mongo.py
+
+Finally, switch to root user as mininet will only run as root and when sudo is used it doesn't have the required dependencies to run kytos and the tests::
+
+  $ sudo su -
+
+Make sure to utilize::
+
+  $ ./kytos-init.sh
+
+To have the correct settings in order for the tests to run properly!
```

Comment:
@HeriLFIU, excellent updated README.rst. For completeness if you could also cover these points below:

- Let's also instruct developers create a new venv and install these requirements https://github.com/amlight/kytos-docker/blob/master/Dockerfile#L60-L65 (equivalent from the Dockerimage), so feel free to add a requirements.txt file:

```
pytest-timeout==2.1.0
pytest==7.2.1
pytest-rerunfailures==10.2
mock==4.0.3
pymongo==4.1.0
requests==2.27.0
```

This new venv would be exactly the same as kytos venv, except it would install these extra testing dependencies here, so having another one would be beneficial not to mix these extra dependencies and causing other surprises when regenerating dependencies. And then, to run the tests it would be from the kytos repo, but then targeting wherever this kytos-end-to-end-tests repo is at, something like:


```
python3 -m pytest --timeout=180 <this_repo_path> -k <expr>
```

Or alternatively, developers can also install these extra deps on the existing `kytosd` venv, but then they'll need to always recreate it when having to generate or introduce new kytos core dependencies (which isn't that frequent), but developers can also do it as long as they're aware of this potential problem. 

Let me know if you could also explore this part, make sure it works on your machine (it's working on mine too) and document on readme the first approach, or let me know if you have any other better suggestions to make this easier.

Reply:
It'd be great to also explain how developers can visualize `kytos.log`, since it can be very handy when fixing or analyzing errors as you know.

Reply:
I added some of the additional info regarding the new virtual environment and its dependencies. I made two requirement files one for local and one for docker. I think it would be best to have two separate environments, but I have been using a single venv for both and as of now I haven't encountered any problems.

Reply:
@HeriLFIU the new commits look good, and well documented too. However, `pip`, `wheel` and `setuptools` aren't typically recommend to be pinned on requirements since they're system/packaging dependencies, although a new venv a being used and consequently it wouldn't result in conflicts with system packages, but still, they're not usually pinned, so if you could remove them, that'd be great. That said, I think in this case only a single requirements file should suffice.